src/main/kotlin/com/atlassian/performance/tools/report/jstat/JstatGcutilParser.kt

```diff
@@ -15,7 +15,7 @@ internal class JstatGcutilParser {
     ): List<SystemMetric> {
         val parser = CSVParser(
             inputStream.bufferedReader(),
-            CSVFormat.DEFAULT.withHeader(JstatGcutilHeader::class.java)
+            CSVFormat.DEFAULT.withFirstRecordAsHeader()
```

Comment:
I'm glad that the CSV-producing code `JstatConverter` is near the CSV-consuming code. This way you could add the CSV header without breaking behavioral compatibility, ie. we still work with old jstat logs.